src/melee/ft/chara/ftDonkey/ftdonkey_SpecialN.c

```diff
@@ -30,7 +30,13 @@ void ftDonkey_UpdateDKVelocityAfterPunch(HSD_GObj* fighter_gobj)
 void ftDonkey_SpecialN_StartAction(HSD_GObj* fighter_gobj)
 {
     Fighter* fp = GET_FIGHTER(fighter_gobj);
-    ftDonkeyAttributes* donkey_attr = getFtSpecialAttrs(fp);
+    ftDonkeyAttributes* donkey_attr = fp->x2D4_specialAttributes;
+
+    /// @todo Unused stack.
+#ifdef MUST_MATCH
+    u8 unused[8];
+#endif
```

Comment:
Why'd these get changed

Reply:
There's something wrong with that inline that causes `GET_FIGHTER` to not match in cases where it should. Given that there are so few exceptions to its usage in `fighter.c`, which does not access special attributes, I think it should be deprecated until we figure out attributes better (including state variable union hell).